CTFd/config.py

```diff
@@ -269,4 +273,8 @@ class TestingConfig(ServerConfig):
 # Actually initialize ServerConfig to allow us to add more attributes on
 Config = ServerConfig()
 for k, v in config_ini.items("extra"):
+    # We should only add the values that are not yet loaded in ServerConfig.
+    if hasattr(Config, k):
+        continue
```

Comment:
I wonder if perhaps this should raise an Exception instead of silently allowing it.